finance/bitcoin/Portfile

```diff
@@ -22,23 +22,31 @@ master_sites            ${homepage}bin/bitcoin-core-${version}/
 checksums           rmd160  a11078445daad75bbe2032da307d292670eece55 \
                     sha256  34de2dbe058c1f8b6464494468ebe2ff0422614203d292da1c6458d6f87342b4
 
-depends_build           port:pkgconfig \
-                        port:autoconf \
-                        port:automake \
-                        port:libtool
-
-depends_lib             port:boost \
-                        path:lib/libssl.dylib:openssl \
-                        port:db48 \
-                        port:libevent \
-                        port:miniupnpc \
-                        port:protobuf-cpp \
-                        port:zmq
+depends_build	port:autoconf port:automake port:libtool port:pkgconfig
+depends_lib	port:boost port:libevent port:miniupnpc port:openssl \
+		port:protobuf-cpp port:python36 port:qt5 port:zmq
 
 use_parallel_build      yes
 
-configure.args          --disable-ccache \
-                        --disable-silent-rules
+configure.args          --disable-ccache		\
+			--disable-silent-rules		\
+			--disable-upnp-default		\
+			--enable-tests			\
+			--disable-bench			\
+			--disable-hardening		\
+			--disable-reduce-exports	\
+			--disable-lcov			\
+			--disable-glibc-back-compat	\
+			--disable-experimental-asm	\
+			--enable-zmq			\
+			--enable-man			\
+			--disable-debug			\
+			--enable-werror			\
```

Comment:
Are you sure you want to enable `-Werror`? Usually we want to disable it, because often a new version of Xcode will contain a new version of the compiler which now presents warnings for things the previous compiler version didn't warn about, which due to `-Werror` turns into errors that prevent building. `-Werror` is great for developers to use, but not so great for users. See https://blog.flameeyes.eu/2009/02/future-proof-your-code-dont-use-werror/

Reply:
Yes, that's intentional. If it fails to build with some future compiler, I want see it fail.
If it happens to be some bogus warning turned into an error, I am willing to remove it,
but unless this actually prevents it from being built with some compilers,
I want it to be as strict as possible.